cmd/minikube/cmd/start.go

```diff
@@ -1226,7 +1226,7 @@ func validatePorts(ports []string) error {
 			if p > 65535 || p < 1 {
 				return errors.Errorf("Sorry, one of the ports provided with --ports flag is outside range %s", ports)
 			}
-			if p < 1024 && i == 0 {
+			if detect.IsMicrosoftWSL() && p < 1024 && i == 0 {
```

Comment:
this means after trhis PR u can use privileged  ports on Non WSL ? is this intentional?

Reply:
Correct, the initial issue was to only add the privilege port validation to WSL but it got added to everything instead. Users are bringing up the issue as now validation is not allowing them to use ports that were previously working.

Reply:
are those users Docker on Linux ? or docker on windows too?

Reply:
For sure Docker Linux, no reports from Docker Windows